### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by mathieu-aubin**
Adds informations about release being available on Fedora 28+
current version on Fedora is 2.4.0 tho.
i will create a rpm spec file to package your release

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -19,6 +19,9 @@ Use the standard Python installation method::
 
   pip3 install git-pull-request
 
+Altho it might not be up to date with the `latest code on GitHub <https://github.com/jd/git-pull-request>`_, `Fedora <https://getfedora.org>`_ 28+ users can also download straight via package manager `DNF <https://fedoraproject.org/wiki/DNF>`_::
```
Comment (jd) [inline README.rst]: Although?

Comment (mathieu-aubin) [reply README.rst]: oh, yes.
pardon my french.

**Comment by mathieu-aubin** [comment]
on the topic of rpms... i have created the specfile and have built rpms for both fedora 28 and 29 (unsigned).

As i am not familiar with pypi and its packaging... i wonder if there's such a file like .pyignore that has the same function has .npmignore (which prevents certain files from being packages in the final distributable npm package)

here's an example of the build running on one of my machines

[asciicast](https://asciinema.org/a/MldqOnAN49Z0kZb1Lm6gFavHB)

also, just for kicks, do you speak french?

Thanks

**Comment by jd** [comment]
`setup.py` should only install what's needed so there's no ignore file AFAICT.

And I'm French. :)

**Comment by mathieu-aubin** [comment]
oh thats great. When i get some free time i'll push my specfile / rpms

Et je suis Québecois, voilà.

I Don't know where else we can communicate really but i have found (maybe) an issue while installing... it seems gitlab wants api V4 now instead of V3. After changing the main file to reflect, i didn't seem to have an error on initialization.

One other thing that is inconvenient... for package automation... where can we get any kind of version number for the project? I mean... is there a way to programatically get a version number (can't seem to find a file with a version number anywhere in this project and gh tags are unreliable at best)

anyways, merci vieux

**Comment by jd** [comment]
`You can use python setup.py --version` to get a version number.

**Pull request merged**